src/CryptoAxus.API/Controllers/ArtistController.cs

```diff
@@ -122,6 +124,29 @@ public async Task<IActionResult> PostArtist([FromBody] CreateArtistDto artist)
             return BadRequest(response);
         return CreatedAtRoute("GetArtistById", new { id = response.Result?.Id }, response);
     }
+    /// </summary>
+    /// <param name="userWalletAddress" example="0x507f191e810c19729de860ea"></param>
+    /// <response code="200">Success response with 200 code and information message about update</response>
+    /// <response code="404">Not Found response with 404 code and information message</response>
+    /// <response code="400">Bad Request response with 400 code and information message</response>
+    /// <returns></returns>
+    [HttpGet("{userWalletAddress:required}/userWalletAddress", Name = "GetArtistByWalletAddressRequest", Order = 5)]
```

Comment:
`userWalletAddress` data type missing in route template

Reply:
Resolved